frontend/src/component/common/Table/SortableTableHeader/CellSortable/CellSortable.styles.ts

```diff
@@ -54,6 +54,7 @@ export const StyledButton = styled('button', {
     ...(isSorted && {
         fontWeight: theme.typography.fontWeightBold,
     }),
+    contain: 'paint',
```

Comment:
That line.
https://developer.mozilla.org/en-US/docs/Web/CSS/contain

Everything else is a refactor I did when familiarizing myself with this part of the code.